keel-sql/src/main/resources/db/changelog/20200925-environment-related-indices.yml

```diff
@@ -11,8 +11,6 @@ databaseChangeLog:
             name: name
         - column:
             name: type
-        - column:
-            name: version
```

Comment:
Will be good to find a solution to this, but since this is a new query and the original PR was designed to tweak performance of existing queries we can live without it for now